src/components/map-component.tsx

```diff
@@ -60,66 +110,120 @@ export class MapComponent extends BaseComponent<IProps, IState>{
+          position={[mapPos.lat, mapPos.lng]}
+          icon={cityIcon}
+          key={name}>
+            <Popup>
+              {name}
+            </Popup>
+          </Marker>
+        );
       }
     });
 
-    const volcanoPos = this.toCanvasCoords({x: volcanoX, y: volcanoY}, gridSize);
+    const { crossPoint1Lat, crossPoint1Lng, crossPoint2Lat, crossPoint2Lng } = this.stores;
+    const volcanoPos = L.latLng(volcanoLat, volcanoLng);
+    const corner1 = L.latLng(volcanoLat - 15, volcanoLng - 15);
+    const corner2 = L.latLng(volcanoLat + 15, volcanoLng + 15);
+    const bounds = L.latLngBounds(corner1, corner2);
+    let viewportBounds = bounds;
+    let mapRef = null;
+    if (this.map.current) {
+      mapRef = this.map.current.leafletElement;
+      viewportBounds = this.map.current.leafletElement.getBounds();
+    }
 
     return (
-      <CanvDiv ref={this.ref}>
-        <Stage
-          width={width}
-          height={height}
-          options={
-            {
-              backgroundColor: Color("hsl(0, 10%, 95%)").rgbNumber(),
-              antialias: true
-            }
-          } >
-          <Sprite image={map} x={0} y={0} width={width} height={height} />
-          <PixiTephraMap
-            canvasMetrics={this.metrics}
-            gridColors={getSnapshot(gridColors)}
-            gridValues={getSnapshot(gridValues)}
-            toCanvasCoords={this.toCanvasCoords} />
+      <CanvDiv
+        ref={this.ref}
+        width={width}
+        height={height}
+      >
+        <LeafletMap
+          className="map"
+          ref={this.map}
+          ondragend={this.reRenderMap}
+          onzoomend={this.reRenderMap}
+          center={[volcanoLat, volcanoLng]}
+          zoom={8}
+          maxBounds={bounds}
+          maxBoundsViscosity={1}
+          minZoom={6}
+          maxZoom={10}
+          attributionControl={true}
+          zoomControl={true}
+          doubleClickZoom={true}
+          scrollWheelZoom={true}
+          dragging={!showCrossSectionSelector}
+          animate={true}
+          easeLinearity={0.35}
+          >
+          <Marker
+            position={[volcanoLat, volcanoLng]}
+            icon={iconVolcano}>
+            <Popup>
+              Popup for any custom information.
+            </Popup>
+          </Marker>
           {cityItems}
-          <PixiAxis gridMetrics={this.metrics} toCanvasCoords={this.toCanvasCoords} />
-          <PixiGrid gridMetrics={this.metrics} />
-          <Sprite image={"./assets/volcano.png"}
-            x={volcanoPos.x - 10}
-            y={volcanoPos.y - 10}
-            width={60}
-            height={60} />
-          <VolcanoEmitter
-            config={AshConfig.config as unknown as EmitterConfig}
-            imagePath={AshConfig.image}
-            x={volcanoPos.x + 20}
-            y={volcanoPos.y + 20}
-            windDirection={windDirection}
+          <TileLayer
+              url="https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png"
```

Comment:
We have a tile server to cache earthquake data (since USGIS database occasionally pops offline during gov. shutdowns) but we do not use a tile server for the map tiles themselves. The OSM tiles for street maps we use in Seismic Explorer also include an optional parameter for scale that comes from window.devicePixelRatio - on Retina displays, map tiles will appear fuzzy unless you add that parameter (@2x) works well if I recall correctly. 

Example usage here: https://github.com/concord-consortium/seismic-explorer/blob/master/js/map-layer-tiles.js. 

Result is seen https://maps.wikimedia.org/osm-intl/8/66/118.png compared with https://maps.wikimedia.org/osm-intl/8/66/118@2x.png - when rendered on retina, that higher resolution image looks super sharp. Let me know if you need a hand here!

Reply:
As an addendum to that - you can use the 2x images on all devices without negative rendering effects - the bandwidth used will be a little higher but it's not a huge map so probably not a big issue. Actual devicePixelRatios can vary, so you could switch to using 2x on any ratio greater than 1